http/src/http.ml

```diff
@@ -1,5 +1,7 @@
 module Transfer = struct
   type encoding = Chunked | Fixed of int64 | Unknown
+
+  let compare_encoding (x : encoding) (y : encoding) = Stdlib.compare x y
```

Comment:
Is this just to expose encoding’s comparison in a type enforcing way or some kind of specialization here also helps speed the code up?

Reply:
No, there's no specialized comparison generated. It's just to prevent things like `Transfer.compare_encoding 42 24`